pkg/cmd/fetch.go

```diff
@@ -16,46 +16,37 @@ package cmd
 
 import (
 	"errors"
+	"fmt"
 
 	"github.com/spf13/cobra"
 
+	reference "github.com/tetratelabs/getenvoy/pkg"
 	"github.com/tetratelabs/getenvoy/pkg/binary/envoy"
-	"github.com/tetratelabs/getenvoy/pkg/manifest"
+	"github.com/tetratelabs/getenvoy/pkg/binary/envoy/globals"
 )
 
 // NewFetchCmd create a command responsible for retrieving Envoy binaries
-func NewFetchCmd() *cobra.Command {
+func NewFetchCmd(o *globals.GlobalOpts) *cobra.Command {
 	return &cobra.Command{
 		Use:   "fetch <reference>",
 		Short: "Retrieve Envoy binaries from GetEnvoy.",
 		Long: `
 Retrieves the referenced Envoy binary from GetEnvoy. The reference can be a full or partial reference.
 A complete list of available builds can be retrieved using` + "`getenvoy list`" + `.`,
-		Example: `# Fetch using a partial manifest reference to retrieve a build suitable for your operating system.
-getenvoy fetch standard:1.11.1
+		Example: fmt.Sprintf(`# Fetch using a partial manifest reference to retrieve a build suitable for your operating system.
+getenvoy fetch %[1]s
```

Comment:
same here

Reply:
same value is used twice. hard to see because there's an extra newline